src/ll/reply.rs

```diff
@@ -47,4 +49,34 @@ impl Response {
         }
         f(&v)
     }
+
+    // Constructors
+    pub(crate) fn new_empty() -> Self {
+        Self::Error(0)
```

Comment:
`Data([])` might be more clear than `Error(0)`, since this is not an error

Reply:
I did it this way as it will be mildly more efficient - we don't need to faf around with `SmallVec` in this case.